Start of theater app

young-george opened the pull request:
Created the events section, footer, and a image placeholder for the image slider

Quinton-H committed:
Start of theater app

Created the events section, footer, and a image placeholder for the image slider

young-george commented on index.html:
```diff
@@ -0,0 +1,102 @@
+
+
+<body>
+    <!--Make a header called Theater App name and a nav bar with different options user can click-->
```
I feel like these comments are a bit superfluous since the code already tells us what it is

Quinton-H replied:
This comment was made by alee she's working on the navbar, I don't know if I delete it will it still be on hers?

young-george replied:
Yes it will still be on hers until the two branches are merged to main

young-george commented on index.html:
```diff
@@ -0,0 +1,102 @@
+                    <li class="navbarlist"><a href="browse.html"></a>Browse Movies</li>
+                    <!--Make 'get tickets' a drop down-->
+                    <li class="navbarlist">Get tickets</li>
+                        <li class="navbar_dropdown_content">
```
choose a consistent css class naming scheme. Some classes are like navbarlist, some are like navbar_dropdown_content, some are like img-slider. Choose between using - or _ to separate words and stick to it throughout the project

ConnorSmith-pf replied:
I would recommend looking at [BEM](http://getbem.com/introduction/), for CSS naming

Quinton-H replied:
We'll have to come up with a name scheme that we agree upon, mine is "img-slide", alee is "navbarlist".

young-george commented on index.html:
```diff
@@ -0,0 +1,102 @@
+    <main>
+        <!-- Image slider using js -->
+        <div class="img-slider">
+          <img class="images" src="https://www.comingsoon.net/assets/uploads/gallery/mortal-kombat/mortal-kombat-poster.jpg" alt="mortal kombat poster">
```
The image looks a bit funky on my screen because it's not maintaining the aspect ratio.
![image](https://user-images.githubusercontent.com/16202746/117639728-590cde80-b1d8-11eb-8a10-8ebdb6176c0b.png)

Quinton-H replied:
I wasn't sure how I sure how I should set the width of it so I just gave it a min-width so that it wouldn't shrink to far down. I'm guessing I should do a max as well

young-george replied:
You can probably also set it up so that the height changes as well while keeping the same aspect ratio

young-george commented on index.html:
```diff
@@ -0,0 +1,102 @@
+            <h1>Events Going on:</h1>
+            <p>National Free Popcorn Day</p>
+            <span>8/24/2021</span>
+          <br>
```
Can probably avoid using <br> here by wrapping each event in a div & adding some styles

Quinton-H replied:
So your saying wrap up the '<h1>', '<p>', and '<span>' in there own divs to add styles

young-george commented on style.css:
```diff
@@ -0,0 +1,179 @@
+    justify-content: space-between;
+  }
+  
+  footer-column {
```
missing . in front of footer-column

Quinton-H replied:
Deleted this since its not needed

young-george commented on style.css:
```diff
@@ -0,0 +1,179 @@
+      margin: 0 auto;
+  }
+  
+  .move-over {
```
don't think this class is used anywhere so can be removed

Quinton-H replied:
removed this

young-george commented on style.css:
```diff
@@ -0,0 +1,179 @@
+  // }
+  
+  /* media queries */
+  @media screen and (max-width: 340px){
```
Can you please explain why each media query is for a different max-width? I feel like these could all just be put in one media query

Quinton-H replied:
I want to say I did this because at different points things would start to look ugly so I broke it up plus the font-size would either shrink the header to much or the paragraphs not enough

young-george commented on style.css:
```diff
@@ -0,0 +1,179 @@
+  }
+  
+  .main-wrapper {
+      width: 90%;
```
Careful using % width here because sometimes 90% width is just too much (like what if they have an ultra wide monitor)

Quinton-H replied:
Sarukei made this, what should it be changed to?

Quinton-H committed:
New update

Quinton-H committed:
Updated some of the css and spacing

Got rid of <br> tags, increased the space between the margins, and downsized the events image

Quinton-H committed:
Update style.css
